starter-code/index.html

```diff
@@ -2,30 +2,85 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+  
+    <div class="background">
+     <img src="./images/bgtile.png" alt="">
+    </div>
+  
+    <div class="mainDiv">
+ 
+        <div class="frames">
+            <section>
+              <img src="./images/phones.png" alt="phone-frames">
+            </section>
+         
+           </div>
+           <div class="container">
+         
+               <section> 
+                 <img src="./images/brand.png" alt="instagram logo">
+                 <img id="home" src="./images/home.png" alt="home"> 
+               </section>
+               
+         
+               <section class= "text">
+                   <h1>Capture and Share <br> the World's Moments
+                     </h1>
+                   <p>
+                       Instagram is a <span>fast, beautiful</span> and <span>fun</span> way to share your life with friends and family.
+                     <br>
+                     <br>
+                       Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+                     <br>
+                     <br>
+                       Oh yeah, did we mention it's free?
+                   </p>
+                   <div class="buttons">
+                      <img id="ios" src="./images/badge-iphone.png" alt="ios">  
+                      <img id="andro" src="./images/badge-android.png" alt="android">
+                     </div>
+               </section>
+               
+               
+               
+               
+               
+         
+           </div>
 
-  Capture and Share the World's Moments
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+    </div>
+  
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
 
-  Oh yeah, did we mention it's free?
+  
+  
+  
+  
+  <footer>
+    <div class="foo">
+        <ul class="footer-links">
+            <li><a href="#">ABOUT US</a></li>
+            <li><a href="#">SUPPORT</a></li>
+            <li><a href="#">BLOG</a></li>
+            <li><a href="#">PRESS</a></li>
+            <li><a href="#">API</a></li>
+            <li><a href="#">JOBS</a></li>
+            <li><a href="#">PRIVACY</a></li>
+            <li><a href="#">TERMS</a></li>
+            &copy; 2014 Instagram
```

Comment:
Don't forget to put you element inside a tag otherwise he won't respond to any of your css rules.